src/components/Common/index.ts

```diff
@@ -0,0 +1,76 @@
+export { AdBanner } from './AdBanner';
+export { Button, buttonVariants } from './Button';
+export {
+  Carousel,
+  type CarouselApi,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+} from './Carousel';
+export { Chatting } from './Chatting';
+export {
+  ErrorBoundary,
+  ErrorFallback,
+  ErrorHandlingWrapper,
+} from './ErrorBoundary';
+export { Footer } from './Footer';
+export {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  useFormField,
+} from './Form';
+export { GameListCard } from './GameListCard';
+export { GameListCarousel } from './GameListCarousel';
+export { GamePanel } from './GamePanel';
+export { GameRoom } from './GameRoom';
+export { HomeClient } from './HomeClient';
+export { InitialNickname } from './InitialNickname';
+export { Input } from './Input';
+export { Label } from './Label';
+export { Logo } from './Logo';
+export { MainHeader, Nicknamebar } from './MainHeader';
```

Comment:
```suggestion
export { MainHeader, NicknameBar } from './MainHeader';
```
다른 대소문자는 괜찮은데 여기서만 소문자로 쓰신 것 같아요. suggestion 남겨둘테니 바로 반영해주시면 될 것 같습니다

Reply:
오 감사합니다 수정하겠습니다!

Reply:
다시 살펴보니 NicknameBar는 MainHeader에서만 사용되는 컴포넌트이기 때문에 export 자체가 불필요하네요. 아예 export 문에서 삭제했습니다!